#!/usr/bin/env python

PACKAGE = "ri_dbscan"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("detect_min",             double_t, 0, "Default: 0.1",    0.1,   0,  2)
gen.add("detect_max",             double_t, 0, "Default: 10",     10,    2,  40)

gen.add("cvc_coef",               double_t, 0, "Default: 1.0",    1.0,    1,  3)

gen.add("MinClusterSize",         int_t, 0,    "Default: 20",      20,   10,   100)
gen.add("MaxClusterSize",         int_t, 0,    "Default: 1000",    10000,   20,   10000)

exit(gen.generate(PACKAGE, "ri_dbscan", "ri_dbscan_"))